src/OpenColorIO/Platform.h

```diff
@@ -73,6 +76,21 @@ void AlignedFree(void * memBlock);
 //       the file if created.
 std::string CreateTempFilename(const std::string & filenameExt);
 
+// Platform-independent ifstream constructor based on UTF-8 input
```

Comment:
We try now to have method explanation starting with a verb and ending with a '.'